docs/data-cookbooks/cnm-workflow.md

```diff
@@ -242,34 +259,36 @@ To add a rule via the dashboard (if you'd like to use the API, see the docs [her
 ```
 
 **Please Note:**
-- The rule - value key must match the Amazon resource name [ARN](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) for the Kinesis data stream you've preconfigured.   You should be able to obtain this ARN from the Kinesis Dashboard entry for the selected stream.
-- The collection and provider should match the collection and provider you setup in the `Prerequisites` section.
 
-Once you've clicked on 'submit' a new rule should appear in the dashboard Rule Overview.
+- The rule's `value` attribute value must match the Amazon Resource Name [ARN](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) for the Kinesis data stream you've preconfigured.   You should be able to obtain this ARN from the Kinesis Dashboard entry for the selected stream.
+- The collection and provider should match the collection and provider you setup in the `Prerequisites` section.
```

Comment:
nitpicky, but I think it would be nice to have prerequisites link to that section like done under the **Sections** header.